lib/double.js

```diff
@@ -41,16 +42,22 @@ class Double {
     if (options && (options.legacy || (options.relaxed && isFinite(this.value)))) {
       return this.value;
     }
-    return { $numberDouble: this.value.toString() };
+
+    if (Object.is(Math.sign(this.value), -0)) {
+      return { $numberDouble: `-${this.value.toFixed(1)}` };
```

Comment:
TIL there's difference between -0 and +0. I think a comment here could be useful to explain this check, it seems pretty obscure.

Reply:
_definitely_ obscure! Okay, I added a comment